data/src/entity_manager/system.py

```
@@ -4962,14 +4988,14 @@ def _process_table_name(self, entity_class, table_name, name):
         # filtering entity and must be considered an exceptional case
         # (only the entity parent class is considered)
         if not table_name:
-            # sets the table name as the single entitie's
+            # sets the table name as the single entity's
             # table name
             table_name = _table_name
         # otherwise in case the entity class is not the current entity
         # class the table name must be created as a composed table name
         elif not _entity_class == entity_class:
```

Ruff (0.11.9)

Use `_entity_class != entity_class` instead of `not _entity_class == entity_class`

Replace with `!=` operator

(SIM201)